frameworks/FSharp/zebra/src/App/App.fsproj

```diff
@@ -1,21 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
```

Comment:
@cartermp oups I copied this from the csproj changes. Is it doing anything in the F#?

Reply:
It will turn on some features, but if they're not used, then no difference.

Bugs and perf improvements are irrespective of LangVersion